documentation/modules/metrics/proc_metrics-deploying-prometheus-alertmanager.adoc

```diff
@@ -1,15 +1,17 @@
 // This assembly is included in the following assemblies:
 //
-// metrics/assembly_metrics-prometheus-alertmanager.adoc
+// metrics/assembly_metrics-prometheus-setup.adoc
 
 [id='proc-metrics-deploying-prometheus-alertmanager-{context}']
 
 = Deploying Alertmanager
 
 [role="_abstract"]
 Use Alertmanager to route alerts to a notification service.
+link:https://prometheus.io/docs/alerting/alertmanager/[Prometheus Alertmanager^] is a plugin for handling alerts and routing them to a notification service.
```

Comment:
I think compoenents might be better name than plugin? AFAIK it runs in a separate process.